boa3/model/builtin/native/gas_methods/balanceofmethod.py

```diff
@@ -0,0 +1,16 @@
+from typing import Dict
+
+from boa3.model.builtin.interop.nativecontract import GasMethod
+from boa3.model.variable import Variable
+
+
+class BalanceOfMethod(GasMethod):
```

Comment:
Since these methods are from NEP17 standard, wouldn't it be better to make them generic?
This way we can reutilize this code when we're going to implement the same method in the NEO interface